frontend/src/utils/theme.js

```diff
@@ -152,12 +179,13 @@ const initTheme = darkMode =>
       },
 
       header: {
-        height: '60px',
+        height: headerHeight,
         scrolledHeight: '84px',
         paddingX: '14px',
         paddingY: '10px',
         boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.25)',
-        bgColor: darkMode ? 'rgba(0, 0, 0, 0.1)' : '#fff',
+        bgColor: darkMode ? 'rgba(0, 0, 0, 0.3)' : 'rgba(255, 255, 255, 0.9)',
+        iconColor: darkMode ? '#4d4d4d' : '#000000',
```

Comment:
Menu and arrow icon buttons should be `'#ffffff'` in dark mode and `'#4d4d4d'` in light mode. And when hovered, they should be `'#ffffff'` in dark mode and `'#000000'` in light mode.